test/nightly/global.resource.test.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ResourceManagementClient } from 'azure-arm-resource';
+import { CosmosDBManagementClient } from 'azure-arm-cosmosdb';
+import { IHookCallbackContext } from 'mocha';
+import * as vscode from 'vscode';
+import { ext, AzureAccountTreeItemWithAttached } from '../../extension.bundle';
+import { longRunningTestsEnabled } from '../global.test';
+import { AzExtTreeDataProvider, createAzureClient } from 'vscode-azureextensionui';
+import { TestAzureAccount } from 'vscode-azureextensiondev';
+
+export let testAccount: TestAzureAccount;
+export let client: CosmosDBManagementClient;
+export const resourceGroupsToDelete: string[] = [];
+
+suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
+    if (!longRunningTestsEnabled) {
+        this.skip();
+    }
+
+    this.timeout(120 * 1000);
```

Comment:
Could you change both of the timeouts to a format like

`this.timeout(10 * 60 * 1000);`

This is very unreadable.  With the above format, it's easy to tell that the timeout is 10 minutes.